crates/forge/tests/data/contracts/gas_checker.cairo

```diff
@@ -262,6 +262,45 @@ mod GasChecker {
             core::pedersen::pedersen(1, 2);
             core::pedersen::pedersen(1, 2);
             core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
+            core::pedersen::pedersen(1, 2);
```

Comment:
Why these additions?

Reply:
See https://github.com/foundry-rs/starknet-foundry/pull/3416#discussion_r2128164238

Reply:
Did you link the correct comment? How are deprecated syscalls related to this?